packages/rum/test/bundle/bundle.spec.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+const { join } = require('path')
+const webpack = require('webpack')
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
+const rimraf = require('rimraf')
+
+const DIST_DIR = join(__dirname, 'dist')
+
+function runWebpack(config, callback) {
+  const compiler = webpack(config)
+  compiler.run((err, stats) => {
+    if (err) {
+      callback(err)
+    }
+    const { errors } = stats.toJson('errors-only')
+    if (errors.length > 0) {
+      callback(errors)
+    } else {
+      callback(null)
+    }
+  })
+}
+
+function getConfig(entry) {
+  return {
+    entry,
+    output: {
+      path: DIST_DIR,
+      filename: 'bundle.js',
+      libraryTarget: 'umd'
+    },
+    mode: 'none',
+    plugins: [new UglifyJSPlugin({})]
+  }
+}
+
+describe('Browser bundle test', () => {
+  afterEach(() => {
+    rimraf.sync(DIST_DIR)
```

Comment:
Instead of this can we change the output directory of the bundle to `tmp/bundle-test`? and also not pollute the dist folder with test temporary files!

Reply:
this dist folder path is inside `test/bundle/dist`.

Reply:
Right, Maybe using `tmp` folder would be a better option still. Since we gitignore that folder, I know this is removing the folder after each test but for example if the process unexpectedly exits!

Reply:
Sure will change the path name